app/src/Controller/Admin/Developer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Dappur\Controller\Admin;
+
+use Dappur\App\FileBrowser;
+use Interop\Container\ContainerInterface;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+/**
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+class Developer extends Controller
+{
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function logs(Request $request, Response $response)
```

Comment:
__[PHPMD]__

The method logs() has a Cyclomatic Complexity of 32. The configured cyclomatic complexity threshold is 10.


<!-- Comment by [Sider](https://sider.review) -->